src/v/cluster/producer_state.h

```diff
@@ -118,17 +130,19 @@ class requests {
     bool operator==(const requests&) const;
     friend std::ostream& operator<<(std::ostream&, const requests&);
 
+    const request_queue& inflight_requests() const {
+        return _inflight_requests;
+    }
+
+    const request_queue& fnished_requests() const { return _finished_requests; }
```

Comment:
typo: fnished -> finished

Reply:
one typo and IDE replicates it everywhere. 🙈